content/en/blog/_posts/2022-10-05-current-state-2019-third-party-audit.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: blog
+title: "Current State: 2019 Third Party Security Audit of Kubernetes"
+date: 2022-10-05
+slug: current-state-2019-third-party-audit
+---
+
+**Authors**: Cailyn Edwards, Pushkar Joglekar, Rory McCune and Rey Lejano
+
+A brand new Third Party Security Audit of Kubernetes is expected to be
+published later this month (Oct 2022). 
+
+In preparation for that, let's look at the state of findings that were made
+public as part of the last [third
+party security audit of
+2019](https://github.com/kubernetes/sig-security/tree/main/sig-security-external-audit/security-audit-2019).
```

Comment:
nit: should we add the Kubernetes version that was used in the 2019 audit
non-blocking since folks can read the 2019 audit

```suggestion
In preparation for that, let's look at the state of findings that were made
public as part of the last [third party security audit of 
2019](https://github.com/kubernetes/sig-security/tree/main/sig-security-external-audit/security-audit-2019)
that was based on Kubernetes 1.13.4.
```